requirements.txt

```diff
@@ -6,3 +6,4 @@ sh
 pytest==4.2.0
 pytest-mock==1.10.1
 requests-mock==1.5.2
+https://github.com/datamade/django-councilmatic/zipball/master
```

Comment:
why not install from the 2.x series on pypi?

Reply:
following the pattern of other dependencies like python-opencivic-data that can affect the models